com.unity.render-pipelines.universal/Editor/2D/GameObjectCreation.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using UnityEditor.SceneManagement;
+using UnityEngine;
+using UnityEngine.Experimental.Rendering.Universal;
+
+namespace UnityEditor.Rendering.Universal
+{
+    static class GameObjectCreation
+    {
+        const int k_PixelPerfectCameraGameObjectMenuPriority = 5;
+
+        [MenuItem("GameObject/2D Object/Pixel Perfect Camera", priority = k_PixelPerfectCameraGameObjectMenuPriority)]
```

Comment:
FYI you have some menu priorities defined on CoreUtils

Reply:
Noted, thanks. The 2d menu priorities are defined in UnityEditor.U2D.MenuItems as part of the 2d package.